README.md

```diff
@@ -56,18 +59,21 @@ gitlabGroup=Group/subgroup
  - **ignore:** This is a comma separated regular expressions to ignore the project names that matches any of these expressions. Optional.
  - **githubOrg:** If GitHub is used as provider, it's possible to define the repository organization in this property. Optional.
  - **gitlabGroup:** If GitLab is used as provider, it's possible to define the group/subgroup in this property. Optional.
+ - **bitbucketTeam:** If BitBucket is used as provider, it's to define your BitBucket Team name in this property. Optional.
 
 ### Note:
  - GitLab only supports token authentication
+ - Bitbucket only supports token authentication
  - Public GitHub url should be the api url ex: api.github.com and not www.github.com
+ - Public Bitbucket url should be the api url ex: api.bitbucket.org and not www.bitbucket.com
 
 ## Enabling and Running
 
 Create a post-commit hook template for Business Central and, finally, start the Business Central with the `org.uberfire.nio.git.hooks` properly set.
 
 ```shell
 $ cd $APP_SERVER_HOME
-$ echo "#\!/bin/bash\njava -jar $APP_SERVER_HOME/hooks/git-push-2.1-SNAPSHOT.jar" > hooks/post-commit
+$ echo "#\!/bin/bash\njava -jar $APP_SERVER_HOME/hooks/git-push-2.2-SNAPSHOT.jar" > hooks/post-commit
```

Comment:
rever the version upgrade changes. I'll do it after the merge myself

Reply:
Done